src/mumble/CMakeLists.txt

```diff
@@ -838,10 +838,17 @@ if(dbus AND NOT WIN32 AND NOT APPLE)
 		PRIVATE
 			"DBus.cpp"
 			"DBus.h"
+			"GlobalShortcut_xdp.cpp"
 	)
 
+	qt_add_dbus_interface(mumble_xdp_SRCS org.freedesktop.portal.GlobalShortcuts.xml globalshortcuts_portal_interface)
+	find_file(PORTALSREQUEST_XML share/dbus-1/interfaces/org.freedesktop.portal.Request.xml PATH_SUFFIXES share PATHS /usr ${CMAKE_INSTALL_PREFIX})
+	qt_add_dbus_interface(mumble_xdp_SRCS ${PORTALSREQUEST_XML} portalsrequest_interface)
+	target_sources(mumble_client_object_lib PRIVATE ${mumble_xdp_SRCS})
```

Comment:
I feel like this should be outsourced into a dedicated cmake function implemented in a file inside the `cmake` directory. Then this place here could simply call that function.

Plus, it seems that we currently don't handle the case when the searched for file can't be found.

Reply:
I moved both files to auxiliary_files.

I am not sure what you mean by adding the function. Do you still want it if we are not looking it up?